### DISCUSSION TITLE
G-Helper causes long login time

### DISCUSSION THREAD

**Discussion by GalatanuM** [seerge/g-helper#2663] [Q&A]
Hello everyone!

I am the proud owner of an Asus TUF A15 507NU laptop and since the update to version 0.170.0 I noticed extended login times after a restart or shutdown. I tried removing G-Helper and saw that the issue is gone. Reinstalled it and the issue is back again, so it has something to do with the app.
Sometimes the laptop becomes unresponsive to any input after logging in.

**Comment by seerge** [comment]
@GalatanuM hello,

What exactly do you understand under "times after restart and shutdown" ? 

G-Helper is just a portable exe that starts after you log in to windows. It just by definition can't affect anything when it's not running.

Do you have [Fast Startup checked in windows](https://www.makeuseof.com/windows-11-turn-on-or-off-fast-startup/) ?
![image](https://github.com/seerge/g-helper/assets/5920850/46c0df8b-18fe-4993-811f-b9b56c61c832)

**Reply by GalatanuM** [reply]
No, I don't have fast startup enabled, but the issue dissapears when g-helper is not installed which is weird. I think it has something to do with the disabling of asus services.

**Reply by seerge** [reply]
@GalatanuM what exactly do you do to "uninstall" g-helper ? do you install AC ? 

You should just check Fast Startup in default balanced power plan, most probably it's a reason of your problems :)

**Reply by seerge** [reply]
@GalatanuM keep in mind that "Fast Boot" is a setting in windows power plan. 

G-Helper would use default windows "balanced" power plan. Fast booth should be enabled there as well, it could be that you have turned it off before by some reason (or some other app did).

And if you do start AC afterwards - it would force it's own power plan where Fast Boot is also enabled.

**Reply by wmd0701** [reply]
Hi! First of all my sincere appreciation for developing ghelper! I am using a flow z13 2025 (395+), and I encounter the same issue, i.e., my boot time noticeably increased after I switched from armoury crate to ghelper. More precisely, the circle rolling animation on boot runs longer. Before switching to ghelper, I already uninstalled armoury crate and asus smart display control, and deactivate all asus services. Considering that ghelper is just an executable I dont get why this happens.

**Reply by seerge** [reply]
As you have correctly noted G-Helper is just an app, it starts after use logon in windows and has no effect on boot.

When did it start booting slowly? After AC uninstall? Or after you have set something in G-Helper?

Do you use any crazy low power limits or so?

**Reply by wmd0701** [reply]
Here is the order of the things: uninstall AC --> reboot --> uninstall asus smart display control --> download & run ghelper --> disable asus services in ghelper. I didnt really notice whether the reboot after uninstall AC was already slower than usual. For the moment I am using ghelper default eco mode, which I guess is using bios default setting.

**Reply by wmd0701** [reply]
Another issue I noticed on my device (z13 2025) is that after I disabled all asus services via ghelper, the system automatically runs into tablet mode (larger UI in taskbar and Edge, poping up screen keyboard on boot, checkboxes in explorer...), although I have keyboard connected. After some trials I found out that it is a service called asusoptimization that determines whether the device should toggle into tablet mode. If I reactivate this service, the weird switch into tablet mode is gone. But screenxpert is back too.

**Reply by seerge** [reply]
@wmd0701 Eco is not default, but it’s just means that dGPU is disabled :) It’s not affecting boot speed.

Try to reset all settings to default and also do Ec-reset and reset windows power plan to all defaults as explained in https://github.com/seerge/g-helper/wiki/Troubleshooting

As for Tablet mode switch, it’s indeed handled by asus optimization service, but G-Helper can also do that. Just run it as admin (and schedule to run ln startup again, so it will also run as admin then)

**Reply by wmd0701** [reply]
just reset everything, boot time doesn't seem to improve. But runing ghelper as admin does avoid tablet switch. Much thanks!

**Reply by seerge** [reply]
@GalatanuM try to disable (or enable) fast boot option in BIOS (F2 on boot) or in windows to see if it makes a difference

**Reply by GalatanuM** [reply]
@seerge I tried that at that time and I didn't notice anything different. What worked for me was a windows reinstall without installing the asus tool that I was asked to install. Everything worked smoothly for me ever since.

**Reply by seerge** [reply]
@GalatanuM sorry, I have mentioned you by an accident :) but since in clean windows everything works just fine (with G-Helper I assume) - it could be that AC uninstaller actually sets something that windows starts booting slowly ?

@wmd0701 can you still try fast boot and also reset windows power plan to defaults https://github.com/seerge/g-helper/wiki/Troubleshooting#resetting-windows-power-plan-to-defaults to see if it makes any difference ?

Alternatively may be install AC (check boot speed?) and uninstall it again. I think in version 6 they have built in uninstaller natively in Add & Remove programs, so you don't need a dedicated one even and can just uninstall it as a regular app.

**Reply by wmd0701** [reply]
I already tried enabling and disenabling fast boot and power plan reset. Sadly they didnt help. I havent tried reinstalling AC yet, because I guess z13 2025 is using a specialized version of AC and dont where to find it.

**Reply by seerge** [reply]
@wmd0701 I don't think there is a special AC version for your device :) It's the same for all the laptops.
